.github/workflows/docs.yml

```diff
@@ -0,0 +1,24 @@
+name: docs
+
+on: [push, pull_request]
+
+jobs:
+  docs:
+    if: github.event.pull_request.head.repo.full_name == github.repository
+    runs-on: ubuntu-latest
+    steps:
+      - name: setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.13
```

Comment:
I'd remove this and let it automatically pick the latest Go version.

Reply:
removed `with`